src/compas_rhino/scene/volmeshobject.py

```diff
@@ -123,33 +128,35 @@ def clear_facelabels(self):
         None
 
         """
-        guids = compas_rhino.objects.get_objects(name="{}.face.*.label".format(self.volmesh.name))  # type: ignore
-        compas_rhino.objects.delete_objects(guids, purge=True)
+        compas_rhino.objects.delete_objects(self._guids_facelabels, purge=True)
 
     # ==========================================================================
     # draw
     # ==========================================================================
 
-    def draw(self, cells=None, color=None):
+    def draw(self):
         """Draw a selection of cells.
 
-        Parameters
-        ----------
-        cells : list[int], optional
-            A list of cells to draw.
-            The default is None, in which case all cells are drawn.
-        color : :class:`compas.colors.Color` | dict[int, :class:`compas.colors.Color`], optional
-            The color of the cells.
-            The default color is :attr:`VolMeshObject.default_cellcolor`.
-
         Returns
         -------
         list[System.Guid]
             The GUIDs of the created Rhino objects.
             Every cell is drawn as an individual mesh.
 
         """
-        self._guids = self.draw_cells(cells=cells, color=color)
```

Comment:
IMO the default `draw` function should draw the faces and not the cells. This would cover faces without cells and avoid creating duplicate faces.